index.js

```diff
@@ -20,92 +20,113 @@ console.log('Up and running on port 3000');
 
 //This object simulates the body of a request from the client
 let post = {
-  memes: [{
-      cell: {
-        url: 'https://prequelmemes.s3.amazonaws.com/3a762d50-55e3-11e7-aed3-2b2e38f1a510',
-        topText: 'This is the image one Top Text',
-        bottomText: 'This is image one bottom text'
-      }
-    },
-    {
-      cell: {
-        url: 'https://prequelmemes.s3.amazonaws.com/1db15eb0-55e3-11e7-aed3-2b2e38f1a510',
-        topText: 'This is image two top text',
-        bottomText: 'This is image two bottom text'
-      }
-    },
-    {
-      cell: {
-        url: 'https://prequelmemes.s3.amazonaws.com/31ed1e50-55e3-11e7-aed3-2b2e38f1a510',
-        topText: 'This is image three top text',
-        bottomText: 'This is image three bottom text'
-      }
-    }
-  ]
+	memes: [{
+		url: 'https://prequelmemes.s3.amazonaws.com/3a762d50-55e3-11e7-aed3-2b2e38f1a510',
```

Comment:
I removed the "cell" naming, didn't seem necessary. If you wanna add it back again, shouldn't be tough to update!